### ISSUE TITLE
Screen readers can read content outside the Touchpoints modal dialog

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5789]
## Area or Component

Static site - touchpoints modal

## Quick Summary
Our touchpoints modal uses `aria-modal` attribute but deque recommends other techniques to hide the content behind the modal because not all screen readers support that attribute.

See which screen readers are not supported: https://a11ysupport.io/tech/aria/aria-modal_attribute

## Accessibility Violation

[deque link](https://docs.deque.com/issue-help/1.0.0/en/reading-order-browse-outside-modal)

## Action Requested

Add aria-hidden="true" attribute to all wrapping elements within <body> that do not contain the modal. You do not need to add aria-hidden="true" to <script> and <style> elements as screen readers do not read content in those tags. Since some assistive technologies do not yet fully support the aria-modal attribute, we recommend using other techniques to hide content outside of a dialog from screen readers.

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by mehansen** [comment]
Closing as won't do - only an issue in Firefox which we don't support

**Comment by mehansen** [comment]
see #4053 and #4082 for more context
any issues related to `reading-order-browse-outside-modal` for modals using react-modal can be ignored
